src/Kygekraqmak/KygekPingTPS/Main.php

```diff
@@ -13,55 +13,56 @@
  *          |____/ |____/                           |_|
  *
  * A PocketMine-MP plugin to see the server TPS and a player's ping
- * Copyright (C) 2020-2021 Kygekraqmak, KygekTeam
+ * Copyright (C) 2020-2021 Kygekraqmak
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
  *
-*/
+ */
 
 declare(strict_types=1);
 
 namespace Kygekraqmak\KygekPingTPS;
 
+use Kygekraqmak\KygekPingTPS\commands\PingCommand;
+use Kygekraqmak\KygekPingTPS\commands\TPSCommand;
+use KygekTeam\KtpmplCfs\KtpmplCfs;
 use pocketmine\plugin\PluginBase;
 use pocketmine\utils\TextFormat;
 
-use KygekTeam\KtpmplCfs\KtpmplCfs;
-
 class Main extends PluginBase {
-	
-	private const IS_DEV = true;
-	public const PREFIX = TextFormat::YELLOW . "[KygekPingTPS] ";
-	private static self $instance;
 
-	public function onEnable() {
-		self::$instance = $this;
-		$this->saveDefaultConfig();
-		/** @phpstan-ignore-next-line */
-        if (self::IS_DEV) {
-            $this->getLogger()->warning("This plugin is running on a development version. There might be some major bugs. If you found one, please submit an issue in https://github.com/KygekTeam/KygekPingTPS/issues.");
-        }
+    /** @var string */
+    public static $prefix = TextFormat::YELLOW . "[KygekPingTPS] " . TextFormat::RESET;
+
+    /** @var self */
+    private static $instance;
+
+    public function onEnable() {
+        self::$instance = $this;
+        $this->saveDefaultConfig();
+
         KtpmplCfs::checkUpdates($this);
-        KtpmplCfs::checkConfig($this, "2.0");
-        
+        KtpmplCfs::checkConfig($this, "1.1");
+
         $this->getServer()->getCommandMap()->registerAll("KygekPingTPS", [new PingCommand($this), new TPSCommand($this)]);
-	}
+        self::$prefix = TextFormat::colorize($this->getConfig()->get("message-prefix", self::$prefix)) . TextFormat::RESET;
+    }
 
-	public static function getInstance() : self {
-		return self::$instance;
-	}
+    public static function getInstance() : self {
+        return self::$instance;
+    }
 
-	public static function replace(string $string, array $replacements = [], bool $default = true) : string {
-	    $defaults = [
-	        "{prefix}" => str_replace("&", "§", self::getInstance()->getConfig()->get("prefix") ?? "[KygekPingTPS]"),
-            "&" => "§"
-        ];
-        
-        $replacements = $default ? array_merge($replacements, $defaults) : $replacements;
-        
-	    return strtr($string, $replacements);
+    /**
+     * @param string $key
+     * @param string $defaultValue
+     * @param string[] $replacements
+     * @return string
+     */
+    public static function getMessage(string $key, string $defaultValue = "", array $replacements = []) : string {
```

Comment:
what is this some sort of magic xd

Reply:
this could product lot of unwanted thing and remove the ability of removing prefix from the returning messages

Reply:
> what is this some sort of magic xd

An easy way to get formatted messages from config.yml with placeholders and prefix.

Reply:
> this could product lot of unwanted thing

Can you explain?

Reply:
> remove the ability of removing prefix from the returning messages

If sometime in the future we need to do this, I can modify the code, since this is intended to be used only for internal purposes.